maven/src/main/java/org/commonjava/maven/galley/maven/GalleyMavenBuilder.java

```diff
@@ -199,7 +200,7 @@ public MavenPluginDefaults getPluginDefaults()
         return pluginDefaults;
     }
 
-    public GalleyMavenBuilder withPluginDefaults( final StandardMaven304PluginDefaults pluginDefaults )
+    public GalleyMavenBuilder withPluginDefaults( final StandardMaven350PluginDefaults pluginDefaults )
```

Comment:
Looks like I made a mistake here, it should be generalized to MavenPluginDefaults.